.github/workflows/tests.yml

```diff
@@ -65,29 +64,34 @@ jobs:
 
       - name: Setup LFS
         uses: ./.github/actions/setup_lfs
-      
+
       - name: Setup environment
         uses: ./.github/actions/setup_env
         with:
           os-label: ${{ matrix.label }}
-      
+
       - name: Install package editable
         if: ${{ !inputs.pip_git }}
         run: |
           pip install -e . --user
-      
+
       - name: Install package git
         if: ${{ inputs.pip_git }}
         run: |
           pip install git+https://github.com/tardis-sn/tardis.git@master
 
+      - name: Install qgridnext
+        if: ${{ !inputs.pip_git }}
+        run: |
+          pip install qgridnext
+
```

Comment:
This shouldn't be needed after the change I suggested in env file. Try removing it

Reply:
I tried removing this but the tests failed because qgridnext is not installed, this code first installs qgridnext for tests to run. If we don't need this the other option would be to enable `pip install -e . --user` line in the file but that will install all the dependencies again.

Reply:
ok